### ISSUE TITLE
Info about polling feature

### ISSUE THREAD

**Issue by KarmaProd** [windkh/node-red-contrib-telegrambot#139]
I am pretty new to GitHub, so please forgive me if this is not an issue. I don't know any better way to ask these questions, please point me to correct way if there's any.

I am using telegrambot on Node-RED on a Raspberry Pi 3B+, on a SIM connection with limited network traffic data.
I would like to know:
1. how much network traffic the polling feature generates, using standard 300ms setting? I only have ONE single Telegram sender node in my flow.
2. why is the polling needed in order to send messages? Isn't it possible to just iniziate a trasmission whenever there's a new message?
3. I am testing 30000ms (i.e. 30 secs) of polling time. Are there any drawbacks? I don't notice any delay in sending the messages.

Of course the question number 2 is the main one.

Thank you

**Comment by windkh** [comment]
Hi I never measured the traffic, but in that case you should make use of a webhook instead of polling.
Sending will always be done immediately, while fetching messages is done by either polling or a callback through the webhook. The readme explains how to set up a webhook...

**Comment by KarmaProd** [comment]
Dear Karl-Heinz, thanks for your time. Please, could you tell me your opinion on my question num. 2? Let me rephrase: since I only need to SEND messages, do you think that could be implemented a node that doesn't need any polling/webhook? I could work on it by myself, just tell me if you think it's feasible.

**Comment by windkh** [comment]
Right now it is not possible as far as I know. However you could directly post your message to the telegram server using conventional HTTP POST nodes. You would have to assemble the complete URL including token and arguments on your own. But that is easy to do. Maybe this helps: https://flows.nodered.org/flow/12b6d3c16ac89634e58be775b6c0bae1

**Comment by KarmaProd** [comment]
The lower-level HTTP POST is a good alternative, thanks for pointing out this simple solution. However, I am testing your Telegram sender node with a setting of 35 minutes polling time (2100000 ms). I believe this is a good trade-off between network traffic and builtin error handling.
